packages/core/contracts/financial-templates/common/TokenFactory.sol

```diff
@@ -24,8 +24,6 @@ contract TokenFactory is Lockable {
         uint8 tokenDecimals
     ) external nonReentrant() returns (ExpandedIERC20 newToken) {
         SyntheticToken mintableToken = new SyntheticToken(tokenName, tokenSymbol, tokenDecimals);
-        mintableToken.addMinter(msg.sender);
```

Comment:
@eng please take a look at this change. this came about after discussing with Nikesh that there is no point really to assign minter and burner to the caller if the caller has the owner role. The owner can set the mint and burn as needed. doing it this way is more conservative as to who we assign these roles to and makes it harder to leave unintended permissions, as was the case in the original factory.